plugin/meshnet.go

```diff
@@ -423,11 +467,29 @@ func cmdDel(args *skel.CmdArgs) error {
 	return nil
 }
 
+func SetInterNodeLinkType() {
+	// TODO: Find a more appropriate (if any) way to figure out intended link type
+	// As of today, daemon gets the intended link type from env INTER_NODE_LINK_TYPE
+	// which is set by deployment file. The daemon further propagates this to plugin
+	// via means of file on host (which is read below) containing the value GRPC or VXLAN
+	b, err := os.ReadFile("/etc/cni/net.d/meshnet-inter-node-link-type")
+	if err != nil {
+		log.Warningf("Could not read iner node link type: %v", err)
+	}
+
```

Comment:
would it make sense to fall back to, say, vxlan in this case?

Reply:
Yes make sense.